tests/perpendicular-flap/README

```diff
@@ -0,0 +1,5 @@
+ccx_preCICE -i flap -precice-participant Solid <-- NOT Calculix, must match yaml.cpp file format
+
+and
+
+SU2_CFD euler...
```

Comment:
A test case! Nice! :heart_eyes: 

Maybe rename this to `README.md`, add a couple of sentences, wrap the code in code blocks, and refer to it from the main `README.md`.

Is this different than the tutorial case? It would make more sense to split this into two PRs: one with the code updates here, and one with the case file updates in the tutorials repository. The reference results can stay here for now, pointing to the tutorial.

One day we will replace this with out system tests, but this will still take a while :see_no_evil: As soon as we define the system for reference results and comparisons, I will move the results there.

Reply:
Missed this, will do! This is same as the tutorial case but noted, let me know if/when would be good to submit a pull-request to the tutorials repo

Reply:
You can already move the test case to a PR updating the tutorials repository, updating the current tutorial.